<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>categoryFormWidget</class>
 <widget class="QWidget" name="categoryFormWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nueva Categoría</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Categoría</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrpción</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryDescriptionLineText">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>180</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryNameLineText">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveCategoryButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelCategoryButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
